docs/mine/lotus/manage-storage-deals.md

```diff
@@ -1,33 +1,139 @@
+
+You can display the miner's current ask price with:
+
+```sh
+lotus-miner storage-deals get-ask
+```
+
+Lotus clients can request the miner price as well with:
 
 ```sh
-lotus-miner storage-deals --help
+lotus client query-ask <minerID>
 ```
 
-For example, `lotus-miner storage-deals set-ask` allows to set the price for storage that your miner uses to respond ask requests from clients.
+## Listing current deals
 
-If deals are ongoing, you can check the data transfers with:
+Current deals and their current state can be obtained with:
 
 ```sh
-lotus-miner data-transfers list
+lotus-miner storage-deals list -v
 ```
 
-:::tip
-Explore the `lotus-miner` CLI. Every command is self-documented and takes a `--help` flag that offers specific information about it.
-:::
+The list displays when the deal was created, the DataCID that is being stored, the wallet address of client that submitted it, the size and the duration in epochs (30 seconds per epoch).
+
+## Using filters to limit deals
+
+Lotus Miners may want to customize on what conditions deals are accepted through custom logic. This can be achieved by providing an external program or script in the `Filter` option of the `[Dealmaking]` [section in the configuration](miner-configuration.md).
+
+If the external program exists with a **success status code (0)**, the deal is accepted. Otherwise, the deal gets rejected.
+
+For example, the following filter only accepts deals from clients with specific addresses:
+
+```sh
+Filter = "jq -e '.Proposal.Client == \"t1nslxql4pck5pq7hddlzym3orxlx35wkepzjkm3i\" or .Proposal.Client == \"t1stghxhdp2w53dym2nz2jtbpk6ccd4l2lxgmezlq\" or .Proposal.Client == \"t1mcr5xkgv4jdl3rnz77outn6xbmygb55vdejgbfi\" or .Proposal.Client == \"t1qiqdbbmrdalbntnuapriirduvxu5ltsc5mhy7si\" '"
+```
+
+The following perl script is another example. It allows to deny expecific clients and to only accept deals that are set to start not too far into the future: [download here](https://gist.github.com/ribasushi/53b7383aeb6e6f9b030210f4d64351d5/9bd6e898f94d20b50e7c7586dc8b8f3a45dab07c#file-dealfilter-pl).
+
+## Blocking content
+
+The Lotus Miner provides tooling to import a DataCID-blocklist:
+
+```sh
+lotus-miner storage-deals set-blocklist blocklist-file.txt
+```
+
+The `blocklist-file.txt` should contain a list of CIDs, separated by newline. The current blocklist can be checked with:
+
+```sh
+lotus-miner storage-deals get-blocklist
+```
+
+To reset and clear the blocklist, run:
+
+```sh
+lotus-miner storage-deals reset-blocklist
+```
+
+## Grouping deals in the same sector
+
+A delay between the moment the deals are received and the start of the sealing of the sector that contains the data allows miners to include multiple deals per sectors when the space permits it. A higher number of deals per sector allows a more efficient operation since it will require less sealing and proving operations.
```

Comment:
```suggestion
A delay between the moment the deals are received and the start of the sealing of the sector that contains the data allows miners to include multiple deals per sector, when space permits it. A higher number of deals per sector allows a more efficient operation since it will require less sealing and proving operations.
```